README.md

```diff
@@ -1512,6 +1513,47 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+### SchemaGraph 
+The `SchemaGraph` is a utility class that encodes a database schema and allows for better schema 
+introspection than the GraphQL schema. The  `SchemaGraph` has three main advantages over the 
+GraphQL schema:
```

Comment:
We can probably drop `over the GraphQL schema` since it's redundant given the previous sentence